From c26cd3ab670a9ff831e9aaddaec06278d67ed106 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Sun, 7 Jul 2019 15:08:07 -0400
Subject: ISL-0.20-add-includes-to-graphite_h

---
 gcc/graphite.h | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/gcc/graphite.h b/gcc/graphite.h
index 7c3ccc09359..25794a32e02 100644
--- a/gcc/graphite.h
+++ b/gcc/graphite.h
@@ -36,7 +36,14 @@ along with GCC; see the file COPYING3.  If not see
 #include <isl/schedule.h>
 #include <isl/ast_build.h>
 
-#ifndef HAVE_ISL_OPTIONS_SET_SCHEDULE_SERIALIZE_SCCS
+#ifdef HAVE_ISL_OPTIONS_SET_SCHEDULE_SERIALIZE_SCCS
+/* isl 0.15 or later.  */
+#include <isl/schedule_node.h>
+#include <isl/id.h>
+#include <isl/space.h>
+
+#else
+/* isl 0.14 or 0.13.  */
 # define isl_stat int
 # define isl_stat_ok 0
 #endif
-- 
